### PR TITLE
Add html and css file to landing page

### PR DESCRIPTION
**Pull request by biruk-bereka**
Add HTML and CSS files to a product landing page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          height="100px"
+          width="100px"
+        >
+        </div>
```
Comment (biruk-bereka) [inline index.html:23]: You're including the logo in the nav tag, but it should only include elements that include navigation elements. And It's required that we enclose this Logo in an anchor <a> tag. Please consider refactoring it. If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
+
+  </head>
+
+
```
Comment (biruk-bereka) [inline index.html:11]: Please remove unnecessary blank space from your codebase. Unnecessary blank space makes your code unclean.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      <h1>Welocme to our product</h1>
+ </div>
+</header>
+
+
+
+
+
+
+  <body>
```
Comment (biruk-bereka) [inline index.html:42]: Please remove unnecessary blank space from your codebase. Unnecessary blank space makes your code unclean.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+       
+      
+       <div id="main" class="container">
+             <div class="jumbtron">
+
+       </div>
+       <div class="flex-container">
+        <div id="home">
+          <h1>Welocme to our product</h1>
+     </div>
```
Comment (biruk-bereka) [inline index.html:52]: Please change the h1 to  h2 since in one HTML there should be one h1 element.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+     
+     <form id="form" action="https://www.freecodecamp.org/email-submit" method="post">
+       <input id='email' type="email" placeholder="Enter a valid email address">
+       <input id="submit" type="submit" >
+      </form>
+      </div>
+
+
+       </div>
```
Comment (biruk-bereka) [inline index.html:80]: Please remove unnecessary blank space from your codebase. Unnecessary blank space makes your code unclean.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+      display: -ms-inline-flexbox;
+      justify-content: center;
+      align-items: center;
+  }
+  
+  #main {
+      margin-top: 50px;
+  }
+  
+
```
Comment (biruk-bereka) [inline styles.css:14]: Please remove unnecessary blank space from your codebase. Unnecessary blank space makes your code unclean.

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  #form {
+    justify-content: center;
+    align-items: center;
+  }
+  
+  #email {
+    background-color:azure;
+  }
+  
+
```
Comment (biruk-bereka) [inline styles.css:46]: Please remove unnecessary blank space from your codebase. Unnecessary blank space makes your code unclean.

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+  
+  
+    #home {
+        text-align: center;
+        height: 50vh;
+    }
+  
+    #contact {
+        height: 50vh;
+    }
```
Comment (biruk-bereka) [inline styles.css:54]: Add one bank space.

**Review by biruk-bereka — COMMENTED** [review]
# Required Changes ♻️ 
 
- [ ] Please see the inline comments below and make the required changes 😃
- [ ] You have not added a summary to the changes you introduced in the PR body. However, please add a summary and make that summary even more descriptive and detailed.

Submit for re-view when you are done with changes.

Thanks

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @biruk-bereka, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

### To Highlight:

- [x] Nice review comments ✔️ 

### OPTIONAL :

- [ ] Please try to be more polite while giving a review. Use words like `kindly`, `please`, etc.

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**